packages/teams-js/src/public/stageView.ts

```diff
@@ -34,6 +34,11 @@ export namespace stageView {
      */
     threadId: string;
 
+    /**
+     * The messageId identifies a particular channel meeting within the channel as specified by the threadId above.
```

Comment:
What is the result of setting messageId? Why would an app developer want to set it? It's good that you document what this field needs to be, but it would be helpful to also document what different/changed behavior setting this property causes.

Reply:
Setting the messageId would be only for opening the stage in a channel meeting context.

Reply:
There is no changed behavior as such. Current implementations will continue to work. 

messageId is needed to make sure that in addition to existing functionalities we can open the stage in a channel meeting context as well. (which was not supported before)